lite/examples/object_detection/ios/ObjectDetection/Extensions/CVPixelBufferExtension.swift

```diff
@@ -26,8 +26,9 @@ extension CVPixelBuffer {
     let imageHeight = CVPixelBufferGetHeight(self)
 
     let pixelBufferType = CVPixelBufferGetPixelFormatType(self)
-
-    assert(pixelBufferType == kCVPixelFormatType_32BGRA)
+    
```

Comment:
Can you remove the whitespaces on this line?

Reply:
good catch. updated